src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -397,12 +397,9 @@ public function get(): ?array
         }
 
         foreach ($messages[$this->stream] ?? [] as $key => $message) {
-            $redisEnvelope = json_decode($message['message'], true);
-
             return [
                 'id' => $key,
-                'body' => $redisEnvelope['body'],
-                'headers' => $redisEnvelope['headers'],
+                'message' => $message,
```

Comment:
Not sure, but I personally would use here another word then `message`. I would go then here with `content` or `envelope` to avoid the `$message['message']['message']` now its a little bit confusing inside the tests.

Reply:
I agree, it's a bit messy. I've just replaced with `data`

Reply:
Is also fine for me :)